usr.sbin/devinfo/devinfo.c

```diff
@@ -31,46 +32,126 @@
  * Print information about system device configuration.
  */
 
-#include <sys/types.h>
+#include <sys/param.h>
```

Comment:
this should sort after unistd.h. We're inconsistent about whether it should be before or after libxo/xo.h, but there's slightly more code in the tree that uses all sys entries after the normal includes w/o directories.

Reply:
Style(9) seemed pretty clear to me:
"Kernel include files (sys/*.h) come first.  If either <sys/types.h> or <sys/param.h> is needed, include it before other include files." What am I missing? I became a dad a few day ago. Please excuse the pace going forward.